### ISSUE TITLE
Battery Color Stuff

### ISSUE THREAD

**Issue by JoshuaDoes** [d0k3/GodMode9#241]
Hello there (again)!

tl;dr: Can't read battery status due to same colors, would be best if battery indicator was moved to bottom of top screen.

This is a minor issue and doesn't necessarily need to be worked on, but kinda makes my eyes hurt a little bit. The contrast between the safe-level colors and the battery status colors is next to none, and it makes it hard to see the current status of the battery. If the battery indicator could maybe be moved to the bottom of the top screen (even with a percentage next to it or something), it'd be greatly appreciated. I know there's the issue with screen space, but as it currently stands there aren't enough drives that take up the entirety of the space so I suppose not much would be lost.

Thanks in advance, and keep up the great work!

**Comment by d0k3** [comment]
Moving it there really is out of question, but I've got an alternative suggestion.

Can you compile? If so, change the colors here:
https://github.com/d0k3/GodMode9/blob/master/source/godmode.c#L113-L115
https://github.com/d0k3/GodMode9/blob/master/source/godmode.c#L123-L124
... and propose an alternative color scheme. Should also look good and make sense, of course

See here for available colors and also the way to define new ones:
https://github.com/d0k3/GodMode9/blob/master/source/common/ui.h#L29-L52

**Comment by JoshuaDoes** [comment]
I'll experiment with some color schemes I've used before. Will update when I find one that works.

**Comment by JoshuaDoes** [comment]
I've made a pull request with my suggested color edits. It works for me, however they're open for improvement.

**Comment by d0k3** [comment]
Fixed by yourself, thankyou!
